### PR TITLE
Import errors

### PR DESCRIPTION
**Pull request by kelseiv**
Update docs per [DAR 2](https://github.com/influxdata/DAR/issues/2) request:

- Add a new section for _Common -import errors_
- Add a link in _Import data from a file with -import_ to _Common -import errors_ 
- Add note about what to do if you don't see import data (and no import errors are generated)

### CODE REVIEW

File: content/influxdb/v1.7/tools/shell.md [hunk 1]
```diff
@@ -325,6 +329,11 @@ We recommend writing points in batches of 5,000 to 10,000 points.
 Smaller batches, and more HTTP requests, will result in sub-optimal performance.
 By default, the HTTP request times out after five seconds.
 InfluxDB will still attempt to write the points after that time out but there will be no confirmation that they were successfully written.
+* To redirect failed import lines to another file, run the following command:
+
+    ```js
+    influx -import -path=metrics-default.gz -compressed > failures
```
Comment (dgnorton) [inline content/influxdb/v1.7/tools/shell.md]: I think errors will be written to `stderr` and not `stdout`. Test this command if you haven't already. May have to change it to this, which will direct errors to a file named `failures` and regular messages to the screen...
```
influx -import -path=metrics-default.gz -compressed 2> failures
```
or this, which will direct all messages to a file named `output`...
```
influx -import -path=metrics-default.gz -compressed > output 2>&1
```

File: content/influxdb/v1.7/troubleshooting/errors.md [hunk 2]
```diff
@@ -350,3 +350,55 @@ query itself has not hit an interrupt point.
...
 **Resources:**
 [Query management](/influxdb/v1.0/troubleshooting/query_management/)
+
+## Common `-import` errors
+
+Find common errors that occur when importing data in the command line interface (CLI).
+
+1. (Optional) Send import errors to a new file by running the `-import` command with the `failures` option:
+
+        influx -import -path= {filename}.gz -compressed > failures
```
Comment (dgnorton) [inline content/influxdb/v1.7/troubleshooting/errors.md]: Same as above

File: content/influxdb/v1.7/troubleshooting/errors.md [hunk 3]
```diff
@@ -350,3 +350,55 @@ query itself has not hit an interrupt point.
...
+
+**Error:** `partial write: points beyond retention policy dropped={number-of-points-dropped}`
+
+This error occurs when an imported data point is older than the specified retention policy and dropped. Verify the correct retention policy is specified in the import file.
+
+### Unnamed import file
+
+**Error:** `reading standard input: /path/to/directory: is a directory`
+
+  This error occurs when the `- import` command doesn't include the name of an import file. Specify the file to import, for example: `$ influx -import -path={filename}.txt -precision=s`
```
Comment (benbjohnson) [inline content/influxdb/v1.7/troubleshooting/errors.md]: The `- import` should be `-import`. (remove space)

File: content/influxdb/v1.7/tools/shell.md (hunk 1 above)
Comment (timhallinflux) [inline content/influxdb/v1.7/tools/shell.md]: influx -import -path=metrics-default.gz -compressed > <my_filename>
-- this is redirecting stdout...which would look something like this:
```2019/08/01 15:29:21 Processed 0 commands
2019/08/01 15:29:21 Processed 841 inserts
2019/08/01 15:29:21 Failed 0 inserts```
```
influx -import -path=metrics-default.gz -compressed 2> <my_filename>
redirects stderr to the file which then would capture the failed import LPs

you can combine them like
influx -import -path=metrics-default.gz -compressed > <my_output_filename> 2> <my_error_LPs_file>

or combine them into one as @dgnorton suggests.

Comment (kelseiv) [reply content/influxdb/v1.7/tools/shell.md]: thank you, @dgnorton

File: content/influxdb/v1.7/troubleshooting/errors.md (hunk 3 above)
Comment (kelseiv) [reply content/influxdb/v1.7/troubleshooting/errors.md]: thank you, @benbjohnson

File: content/influxdb/v1.7/tools/shell.md (hunk 1 above)
Comment (kelseiv) [reply content/influxdb/v1.7/tools/shell.md]: thank you, @timhallinflux

**Pull request merged**